src/procedures/proc_pagerank.c

```diff
@@ -48,8 +68,8 @@ ProcedureResult Proc_PagerankInvoke(ProcedureCtx *ctx, const SIValue *args) {
 	pdata->i = 0;
 	pdata->g = g;
 	pdata->node = GE_NEW_NODE();
-	pdata->mappings = mappings;
-	pdata->rankings = rankings;
+	pdata->mapping = mapping;
+	pdata->ranking = ranking;
```

Comment:
These will always be assigned at 165-166.

Reply:
Not if we quickly return at lines 83 and 91